### PR TITLE
Refacto panel view

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/19
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1014

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee [hunk 1]
```diff
@@ -0,0 +1,28 @@
+tabViewFormLoad = (options) ->
+  tabViewClass = appConfigurationView.getConfiguration(options.entityType, 'showTab')
+  elementTabViewClass = appConfigurationView.getConfiguration(options.entityType, 'editEntityTab')
+
+  tabView = new tabViewClass(options)
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee]: Couldn't you get the class frow the viewConfigurator?

File: BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  panels = getPanelsLink(options.element.get('links'))
+
+  for panel, i in panels
+    do (panel, i) ->
+      $.ajax
+        url: panel.link
+        method: "GET"
+        success: (response) ->
+          view = new elementTabViewClass(
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee]: same

**Comment by nicolasThal** [comment]
:+1:

**Comment by nicolasThal** [comment]
do not forgot to update the doc according to your modification

File: BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee (hunk 2 above)
Comment (juchi) [inline BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee]: we should be able to have a different view for each tab
@nicolasThal @itkg-alavieille

**Comment by itkg-alavieille** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1017

**Pull request merged**